.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+dist: trusty
+sudo: required
+addons:
+  firefox: latest
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+node_js:
+  - node
+  - 6
+cache: yarn
+install:
+  - yarn global add polymer-cli
+  - polymer install
+before_script:
+  - polymer lint
```

Comment:
I'd leave this out here as it's likely to fail at head. I'll add it in as part of the PR or PRs that get things linting clean.